From f0df5fd0ccb8dbb4ce25dfd04e61fc14cb165975 Mon Sep 17 00:00:00 2001
From: Sangjin Lee <sjlee@apache.org>
Date: Wed, 3 Feb 2016 08:58:49 -0800
Subject: [PATCH 1489/2848] HADOOP-12761. incremental maven build is not
 really incremental (sjlee)

(cherry picked from commit 4dc0a3949386ce2961356143a5a843dd537829dc)

Conflicts:
	hadoop-common-project/hadoop-common/CHANGES.txt
	hadoop-project/pom.xml

Change-Id: Ic09de9365bbaa4feb7f0db112d2f9deed3b22325
---
 hadoop-project/pom.xml |    1 +
 1 file changed, 1 insertion(+)

diff --git a/hadoop-project/pom.xml b/hadoop-project/pom.xml
index 46c59c3..7c1e3c3 100644
--- a/hadoop-project/pom.xml
+++ b/hadoop-project/pom.xml
@@ -986,6 +986,7 @@
           <configuration>
             <source>${javac.version}</source>
             <target>${targetJavaVersion}</target>
+            <useIncrementalCompilation>false</useIncrementalCompilation>
           </configuration>
         </plugin>
         <plugin>
-- 
1.7.9.5

